vhdbuilder/packer/components.json

```diff
@@ -325,7 +325,8 @@
       "downloadURL": "mcr.microsoft.com/oss/azure/aad-pod-identity/nmi:*",
       "versions": [
         "v1.7.0",
```

Comment:
all cluster switched to use 1.7.4 now?
if yes, we can remove the 1.7.0

Reply:
updated, PTAL